src/main/java/com/sericulture/authentication/model/entity/RoleMaster.java

```diff
@@ -0,0 +1,40 @@
+package com.sericulture.authentication.model.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.util.Date;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name="ROLE_MASTER")
+@Getter
+@Setter
+public class RoleMaster {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "role_master_seq")
+    @SequenceGenerator(name = "role_master_seq", sequenceName = "role_master_seq", allocationSize = 1)
+    @Column(name = "role_id")
+    private Long roleId;
+
+    @Column(name = "ROLE_NAME")
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 4, expected level should be 2.